libnetwork/iptables/iptables.go

```diff
@@ -435,14 +435,8 @@ func filterOutput(start time.Time, output []byte, args ...string) []byte {
 // Raw calls 'iptables' system command, passing supplied arguments.
 func (iptable IPTable) Raw(args ...string) ([]byte, error) {
 	if firewalldRunning {
-		// select correct IP version for firewalld
-		ipv := Iptables
-		if iptable.ipVersion == IPv6 {
-			ipv = IP6Tables
-		}
```

Comment:
This logic looks to be to handle empty values / defaults; is that still needed? (i.e., is it guaranteed to be always set?)

Reply:
> is it guaranteed to be always set?

AFAICT, yes it is. `*IPTable` is always instantiated through `GetIptable` which takes a `version` arg used to set the `ipVersion` field.